src/docs/GetStarted/Developers/configureStylelint/index.mdx

```diff
@@ -56,8 +56,8 @@ defaultConfig: {
 ```
 
 <Highlight title="More informations here">
-
-Please checkout [Mozaic configuration](GetStarted/Developers/Configuration/#stylelint-options-) for more information. 
+  
+Please checkout [Mozaic configuration](/GetStarted/Developers/Configuration/#stylelint-options-) for more information. 
```

Comment:
It did'nt work :/

Reply:
I don't understand why, the code looks like good.

Reply:
fixed